README.md

```diff
@@ -10,67 +10,77 @@ This is the development home for Cyberduck, a libre file transfer client for mac
 <img src="https://cdn.cyberduck.io/img/cyberduck-icon-rect-512.png" alt="Cyberduck Icon" width="400px"/>
 
 ## Mailing Lists
+
 There is a [Google Groups Cyberduck](http://groups.google.com/group/cyberduck/) discussion mailing list.
 
 The following additional [mailing lists](http://lists.cyberduck.io/) are hosted:
 
- * [News](http://lists.cyberduck.io/mailman/listinfo/cyberduck-news) Announcements of new releases
- * [Localization](http://lists.cyberduck.io/mailman/listinfo/cyberduck-localization) Notifications about changes to the translations needed
+* [News](http://lists.cyberduck.io/mailman/listinfo/cyberduck-news) Announcements of new releases
+* [Localization](http://lists.cyberduck.io/mailman/listinfo/cyberduck-localization) Notifications about changes to the translations needed
 
 ## Source
-Source code is available licenced unter the [GNU General Public Licence Version 3](https://www.gnu.org/licenses/gpl-3.0.en.html). Pull requests welcome!
+
+Source code is available licensed under the [GNU General Public License Version 3](https://www.gnu.org/licenses/gpl-3.0.en.html). Pull requests welcome!
 
 ## Localizations
+
 Translations to new languages are welcome. We use [Transifex](https://www.transifex.com/cyberduck/cyberduck/dashboard/) to localize resources. Current available localizations are _English_, _Czech_, _Dutch_, _Finnish_, _French_, _German_, _Italian_, _Japanese_, _Korean_, _Norwegian_, _Portuguese_, _Slovak_, _Spanish_, _Chinese (Traditional & Simplified Han)_, _Russian_, _Swedish_, _Hungarian_, _Danish_, _Polish_, _Indonesian_, _Catalan_, _Welsh_, _Thai_, _Turkish_, _Hebrew_, _Latvian_, _Greek_, _Serbian_, _Georgian_ and _Slovenian_.
 
 Make sure to subscribe to the [localization mailing list](http://lists.cyberduck.ch/mailman/listinfo/cyberduck-localization).
 
 ## Support & Documentation
+
 For general help about using Cyberduck, please refer to the [documentation](https://docs.cyberduck.io).
 
 ## Snapshot and Beta builds
-These are nightly snapshot builds from the current development trunk featuring the latest bugfixes and enhancements. Be warned, tough, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
+
+These are nightly snapshot builds from the current development trunk featuring the latest bug fixes and enhancements. Be warned, though, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
 
 ## Prerequisites
 
-- Java 11 SDK or later
-- Apache Ant 1.10.1 or later
-- Apache Maven 3.5 or later
+* Java 11 SDK or later
```

Comment:
Why do you consider dashes deprecated syntax?

Reply:
Following [this standard](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md004), but it is only a suggestion. It's now been reverted.